### PR TITLE
[HMC-111] - Styling Toast

### PR DESCRIPTION
**Pull request by tjoetmuty**
## Description :
- styling toast

## Screenshots / Video Record:
- success
image

- failed
image

### CODE REVIEW

**Review by mpsalunggg — COMMENTED** [review]
Screenshot 2024-03-31 at 06 09 52

I think still different 🤔

**Comment by mpsalunggg** [comment]
Screenshot 2024-03-31 at 06 14 34

Icon success, use this icon, check the documentation lucide react icon

File: src/components/ui/toaster.tsx [hunk 1]
```diff
@@ -9,21 +9,61 @@ import {
...
-          <Toast key={id} {...props}>
-            <div className="grid gap-1">
-              {title && <ToastTitle>{title}</ToastTitle>}
-              {description && (
-                <ToastDescription>{description}</ToastDescription>
-              )}
+          <Toast key={id} {...props} className={IconToast[type].color}>
+            <div
+              className={`flex border-l-2 gap-2 ${IconToast[type].borderColor}`}
+            >
```
Comment (mpsalunggg) [inline src/components/ui/toaster.tsx]: better to use the existing object desctructure

Comment (tjoetmuty) [reply src/components/ui/toaster.tsx]: doneee

File: src/components/ui/toaster.tsx [hunk 2]
```diff
@@ -9,27 +9,27 @@ import {
...
     icon: <Ban className="text-red-500" />,
-    color: 'text-red-500',
+    color: 'text-black',
     borderColor: 'border-red-500'
   },
   success: {
-    icon: <CheckCircle className="text-green-500" />,
-    color: 'text-green-500',
+    icon: <CheckCircle2 className="text-green-500" />,
+    color: 'text-black',
```
Comment (mpsalunggg) [inline src/components/ui/toaster.tsx]: Don't use color key, if they are all the same

File: src/components/ui/toaster.tsx [hunk 3]
```diff
@@ -47,12 +47,10 @@ export function Toaster() {
...
       }) {
-        const { icon } = IconToast[type];
+        const { icon, color, borderColor } = IconToast[type];
         return (
-          <Toast key={id} {...props} className={IconToast[type].color}>
-            <div
-              className={`flex border-l-2 gap-2 ${IconToast[type].borderColor}`}
-            >
+          <Toast key={id} {...props} className={color}>
+            <div className={`flex border-l-2 gap-2 ${borderColor}`}>
```
Comment (mpsalunggg) [inline src/components/ui/toaster.tsx:49]: make the div style slightly rounded, like in figma

File: src/components/ui/toaster.tsx (hunk 2 above)
Comment (tjoetmuty) [reply src/components/ui/toaster.tsx]: check it please a7e5f31b2f348a1658f0d0e4550c03080b40f9f8

**Review by vickyadrii — APPROVED** [review]

**Review by mpsalunggg — COMMENTED** [review]
last remove space

**Review by mpsalunggg — APPROVED** [review]

**Comment by mpsalunggg** [comment]
gg,gg,gg

**Pull request merged**